app.rb

```diff
@@ -7,14 +7,37 @@
   # What are the properties for each of the two types of hashes
 
 # Use enumerables to -
-  # Print all the addresses for the apartments
-  # Print all the names for tenants
-  # Print only apartments that are less then 700 in rent
-  # Print only tenants that are over the age of 44
-  # Print only tenants that have an apartment id of 1
-  # Print all the tenants in order from youngest to oldest
-  # Print the names of all the tenants alphabetically
+  # # Print all the addresses for the apartments
+  # apartments.each{|x| puts x[:address]}
+  # # Print all the names for tenants
+  # tenants.each{|x| puts x[:name]}
+  # # Print only apartments that are less then 700 in rent
+  # apartments.each{|x| puts x[:monthly_rent] < 700}
+  # # Print only tenants that are over the age of 44
+  # tenants.each{|x| puts x[:age] > 44}
+  # # Print only tenants that have an apartment id of 1
+  # tenants.each{|x| puts x[:id] == 1}
```

Comment:
This will print out whether `x[:id] == 1` evaluatues to `true` or `false` -- not the tenant itself. You would need to wrap `puts tenant` in a proper conditional.